friendshipper/src-tauri/src/repo/operations/pull.rs

```diff
@@ -30,9 +31,11 @@ use super::{DownloadDllsOp, RepoStatusRef, StatusOp};
 #[derive(Clone)]
 pub struct PullOp {
     pub app_config: AppConfigRef,
-    pub uproject_path_relative: String,
+    pub repo_config: RepoConfigRef,
+    pub ofpa_cache: unreal::OFPANameCacheRef,
+    // pub uproject_path_relative: String,
     pub repo_status: RepoStatusRef,
-    pub trunk_branch: String,
+    // pub trunk_branch: String,
```

Comment:
can we get rid of these if we don't need em?

Reply:
oops ty